libs/chain-signatures/contract/src/state/mod.rs

```diff
@@ -149,6 +165,15 @@ impl ProtocolContractState {
         .map(|x| x.map(ProtocolContractState::Initializing))
     }
 
+    pub fn vote_code_hash(&mut self, code_hash: CodeHash) -> Result<(), Error> {
+        // TODO verify TEE quote here
```

Comment:
Maybe open an issue for this.

Reply:
Good point! https://github.com/Near-One/mpc/issues/378